engine/markdown/Options/Interactivity/Modes.md

```diff
@@ -8,7 +8,7 @@
 | `grab.distance`              | `number`           | `100`   |                                                        |
 | `grab.lineLinked.opacity`    | `number`           | `0...1` |                                                        |
 | `bubble.distance`            | `number`           | `100`   |                                                        |
-| `bubble.size`                | `number`           | `40`    |                                                        |
+| `bubble.size`                | `number`           | `40`    | Expressed as % increase or decrease of the size of the <br> particle upon mouseover. Note that a value of <br> 100 will result in no change to the particle's size <br> upon mouseover. |
```

Comment:
Is not a percent value, it’s the target value of the particle when the interaction has happened. If the value is less than particle current size the particle will shrink, if it’s greater the particle will grow